setup.py

```diff
@@ -104,6 +104,9 @@
             "langchain",
             "openai",
         ],
+        "mssql": [
+            "pymssql<2.2.8",
```

Comment:
why is this pinned?

Reply:
2.2.8 doesn't install on windows, where I personally develop. Safe to assume we want to ensure platform compatibility?

Reply:
yes, although it seems to me like pymssql should work on any platform. i don't think this should be a constrant of sqlmesh as they may fix the issue. additionally, it could just be your machine, so i don't think we should pin it here, it can be a local override for folks who need it